Sources/Quick/NSString+C99ExtendedIdentifier.swift

```diff
@@ -22,12 +21,25 @@ public extension NSString {
         return invalidCharacters
     }()
 
-    @objc(qck_c99ExtendedIdentifier)
     var c99ExtendedIdentifier: String {
         let validComponents = components(separatedBy: NSString.invalidCharacters)
         let result = validComponents.joined(separator: "_")
 
         return result.isEmpty ? "_" : result
     }
 }
+
+/// Extension methods or properties for NSObject subclasses are somehow invisible
+/// from the Objective-C runtime on static linking, so let's make a wrapper class.
```

Comment:
Because we know that `-ObjC` will expose these to the runtime, this comment might need an update.